core/vpgl/file_formats/vpgl_nitf_RSM_camera_extractor.cxx

```diff
@@ -614,8 +680,10 @@ vpgl_nitf_RSM_camera_extractor::scan_for_RSM_data(bool verbose)
 
         nitf_tre<std::string> nt3("SID", false, true);
         nt3.get_append(tres_itr, tre_str, v);
+        if(nt3.get(tres_itr, mdata.sid_))
+           mdata.sid_valid = true;
 
-        nitf_tre<std::string> nt4("STID", false, true);
+           nitf_tre<std::string> nt4("STID", false, true);
```

Comment:
is this line intended to be part of the `if` block?  (Need braces if so, added intentation is misleading if not)